src/SWP/Bundle/ContentBundle/Tests/Controller/ArticleControllerTest.php

```diff
@@ -52,8 +52,44 @@ public function testLoadingArticleCustomTemplate()
         ]);
         $this->assertEquals(200, $client->getResponse()->getStatusCode());
         $responseArray = json_decode($client->getResponse()->getContent(), true);
-        $this->assertArraySubset(json_decode('{"id":"\/swp\/default\/content\/features","title":"Features","body":"Features content","slug":"features","published_at":null,"status":"new","template_name":"test.html.twig","locale":"en","deleted_at":null,"children":null}', true), $responseArray);
+        $this->assertArraySubset(json_decode('{"id":"\/swp\/default\/content\/features","title":"Features","body":"Features content","slug":"features","status":"published","template_name":"test.html.twig","locale":"en","deleted_at":null,"children":null}', true), $responseArray);
```

Comment:
Better, I think, to test just for the changed element, and not for so much of the response, as it's harder to read, and maintain